src/components/Concentrator/ConcentratorRewards.tsx

```diff
@@ -1,9 +1,87 @@
 import { FC } from "react"
 
-// TODO: Rewards menu
+import { ConcentratorTargetAsset, FilterCategory } from "@/lib/types"
+import {
+  useConcentratorAddress,
+  useFilteredConcentratorVaults,
+} from "@/hooks/data/concentrators"
+
+import Button from "@/components/Button"
+
+type ConcentratorRewardsProps = {
+  concentratorTargetAsset: ConcentratorTargetAsset
+  filterCategory: FilterCategory
+}
+
+export const ConcentratorRewards: FC<ConcentratorRewardsProps> = ({
+  concentratorTargetAsset,
+  filterCategory,
+}) => {
+  // this is pretty convoluted...
+  // get the vaults relevant to this concentrator
+  const vaultsForThisConcentrator = useFilteredConcentratorVaults({
+    concentratorTargetAsset,
+    filterCategory,
+  })
+  // take the first one, because they all have the same vaultAssetAddress
+  const firstVaultForThisConcentrator = vaultsForThisConcentrator.data?.[0]
+  // use the vaultAssetAddress to get the concentrator address
+  // TODO: use concentratorAddress to interact with concentrator contract
+  const _concentratorAddress = useConcentratorAddress({
+    concentratorTargetAsset,
+    vaultAssetAddress: firstVaultForThisConcentrator?.vaultAssetAddress,
+    vaultType: firstVaultForThisConcentrator?.vaultType,
+  })
 
-export const ConcentratorRewards: FC = () => {
   return (
-    <div className="aspect-square rounded-md bg-black/60 backdrop-blur-md"></div>
+    <div className="divide-y divide-pink/30 rounded-md bg-pink-900/80 px-4 backdrop-blur-md">
+      <div className="grid grid-cols-[auto,1fr,auto] items-center gap-2 py-4">
+        <div className="relative h-9 w-9 rounded-full bg-white">
+          {/* TODO: Need logos for target assets */}
+          {/* Should we have those hardcoded? Should we get them by address? How to get address? */}
+          {/* <AssetLogo name={concentratorTargetAsset} className="h-6 w-6" /> */}
+        </div>
+        <div>
+          <h1 className="text-sm">Concentrator</h1>
+          <h2 className="font-semibold">
+            <span className="bg-gradient-to-r from-orange to-pink bg-clip-text text-transparent">
+              {concentratorTargetAsset}
+            </span>
+          </h2>
+        </div>
+        <div>
+          <dl className="text-right">
+            <dt className="text-sm">APY</dt>
+            <dd className="font-semibold">
+              <span className="bg-gradient-to-r from-orange to-pink bg-clip-text text-transparent">
+                23.35%
+              </span>
+            </dd>
+          </dl>
+        </div>
+      </div>
+      <div className="py-4">
+        <dl className="grid grid-cols-[1fr,auto] gap-y-2">
```

Comment:
I took some small liberties with the design here. I believe this to be a bit cleaner:

<img width="296" alt="image" src="https://user-images.githubusercontent.com/95545650/219310720-4e19b2ef-30f7-4fa4-ba38-ec69432f9051.png">